File: app/src/main/java/com/sample/app/activity_tab_pager/fragments/Tab1Fragment.java
package com.sample.app.activity_tab_pager.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.sample.app.R;

public class Tab1Fragment extends Fragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        View rootView      = inflater.inflate(R.layout.fragment_tab1, container, false);
        return rootView;
    }
}

File: app/src/main/java/com/sample/app/activity_tab_pager/fragments/Tab1Fragment.kt
package com.sample.app.activity_tab_pager.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.sample.app.R

class Tab1Fragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {

        val rootView = inflater!!.inflate(R.layout.fragment_tab1, container, false)
        return rootView
    }
}
